src/SpacePrint.jl

```diff
@@ -31,6 +31,13 @@ tab - what happens before this row. Grows with each index.
 final - whether or not this is the final item in the list. Affects the
         corner style. If nothing, no corner or filler.
 """
+function spaceprint(
+    item; index::Int = 1, tab::String = "", corner::String = "", final::Bool = true
```

Comment:
This was a lazy fix, let me know if you have a specific thing in mind (the spaceprint looks ok, you can see the operator as a parameter)